assets/js/modules/analytics-4/components/audience-segmentation/dashboard/InfoNoticeWidget/index.test.js

```diff
@@ -0,0 +1,144 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+	render,
+	provideModules,
+	createTestRegistry,
+	provideModuleRegistrations,
+	provideUserInfo,
+	fireEvent,
+} from '../../../../../../../../tests/js/test-utils';
+import { withWidgetComponentProps } from '../../../../../../googlesitekit/widgets/util';
+import { CORE_USER } from '../../../../../../googlesitekit/datastore/user/constants';
+import InfoNoticeWidget from '.';
+import { AUDIENCE_INFO_NOTICE_SLUG, AUDIENCE_INFO_NOTICES } from './constant';
+import { WEEK_IN_SECONDS } from '../../../../../../util';
+
+describe( 'InfoNoticeWidget', () => {
+	let registry;
+
+	let dismissPromptSpy;
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+		provideModules( registry, [
+			{
+				active: true,
+				connected: true,
+				slug: 'analytics-4',
+			},
+		] );
+		provideModuleRegistrations( registry );
+		provideUserInfo( registry );
+
+		dismissPromptSpy = jest.spyOn(
+			registry.dispatch( CORE_USER ),
+			'dismissPrompt'
+		);
+	} );
+
+	const WidgetWithComponentProps =
+		withWidgetComponentProps( 'InfoNoticeWidget' )( InfoNoticeWidget );
+
+	it( 'info notice should not render when permanently dismissed', async () => {
+		registry.dispatch( CORE_USER ).receiveGetDismissedPrompts( {
+			[ AUDIENCE_INFO_NOTICE_SLUG ]: {
+				expires: 0,
+				count: AUDIENCE_INFO_NOTICES.length,
+			},
+		} );
+
+		const { queryByText, waitForRegistry } = render(
+			<WidgetWithComponentProps />,
+			{
+				registry,
+			}
+		);
+
+		const result = queryByText(
+			'The higher the portion of new visitors you have, the more your audience is growing. Looking at what content brings them to your site may give you insights on how to reach even more people.'
+		);
+
+		await waitForRegistry();
+
+		expect( result ).not.toBeInTheDocument();
+	} );
+
+	it( 'info notice should render when expiry has passed', async () => {
+		const currentTimeInSeconds = Math.floor( Date.now() / 1000 );
+
+		registry.dispatch( CORE_USER ).receiveGetDismissedPrompts( {
+			[ AUDIENCE_INFO_NOTICE_SLUG ]: {
+				expires: currentTimeInSeconds - 10,
+				count: 2,
+			},
+		} );
+
+		const { queryByText, waitForRegistry } = render(
+			<WidgetWithComponentProps />,
+			{
+				registry,
+			}
+		);
+
+		const result = queryByText(
+			'Configure your own custom audiences in Analytics to gain deeper insights into visitor behavior, for example consider creating a “Existing customers” or “Subscribers” segment, depending on what goals you have for your site.'
+		);
+
+		await waitForRegistry();
+
+		expect( result ).toBeInTheDocument();
+	} );
+
+	it( 'info notice should call the onDismiss handler when CTA is clicked', async () => {
```

Comment:
I think we could slightly expand this test case to verify the temporary versus permanent dismissal, based on the dismiss count, i.e. verify that it dismisses for two weeks by default, but when the dismiss count is `AUDIENCE_INFO_NOTICES.length - 1`, it is dismissed permanently.